.github/workflows/custom-actions/build-docs/action.yaml

```diff
@@ -0,0 +1,46 @@
+name: Build Shaka Packager Docs
+
+description: |
+  A reusable action to build Shaka Packager docs.
+  Leaves docs output in the "gh-pages" folder.
+  Only runs in Linux due to the dependency on doxygen, which we install with
+  apt.
+
+runs:
+  using: composite
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: |
+        echo "::group::Install dependencies"
```

Comment:
Just a nit: Can we enable the echoing of the commands instead of using custom messages? I think it is more readable here and it is also more useful for debugging.

Reply:
I tried that first, but I found that for long-running commands with lots of output, groups were more useful in the GitHub UI to review what happened.  Locating a command echoed with set -x in the middle of 10k lines of output is hard.  Finding a group and expanding it is easy.